indexing-service/src/test/java/org/apache/druid/indexing/common/task/IngestionTestBase.java

```diff
@@ -222,11 +224,16 @@ public TaskActionToolbox createTaskActionToolbox()
         taskStorage,
         storageCoordinator,
         new NoopServiceEmitter(),
-        null,
+        supervisorManager,
         objectMapper
     );
   }
 
+  protected void setSupervisorManager(SupervisorManager supervisorManager)
```

Comment:
Rather than this, have a separate `createTaskActionToolbox` method that accepts a `SupervisorManager`.